### DISCUSSION TITLE
能优化一下用量显示吗？

### DISCUSSION THREAD

**Discussion by BAKAOLC** [immersive-translate/immersive-translate#2052] [Help] [answered]
### 概述 | Summary

用量查询的网页虽然在个人中心但是没有在里面有直接进入的方式，为什么会这么设计？
经常会忘记用量查询的页面在哪，又不想污染我自己的收藏夹，能不能优化一下这个？
即使不方便把这个直接显示在插件信息栏里面，好歹让个人中心里面能直接看到用量而不需要我去文档里面找吧？

**Comment by theowenyoung** [comment, marked as answer]
感谢反馈，今天上线了新版的个人中心，目前已在订阅版本提供入口: https://immersivetranslate.com/profile
